app/views/comments/_comment.html.erb

```diff
@@ -0,0 +1,9 @@
+  <div class= "comment">
+    <%comments.each do |comment|%>
```

Comment:
The convention would suggest that a partial called comment would render a single comment rather than a collection.